keylime/cloud_verifier_tornado.py

```diff
@@ -573,6 +573,8 @@ def post(self):
                         logger.info("Allowlist data not provided with request! Using default empty allowlist.")
                         ima_policy = json.dumps(ima.process_ima_policy(ima.EMPTY_ALLOWLIST, []))
 
+                    # Although legit in theory, the following initialization breaks the CI tests
```

Comment:
Instead of putting this here maybe open a ticket about it? I don't like leaving commented out code in and will most likely be forgotten.

Reply:
very good point

Reply:
good point, and I already have a ticket for that, I will simply remove the comment.

Reply:
done